staging/src/k8s.io/legacy-cloud-providers/openstack/openstack.go

```diff
@@ -680,6 +699,11 @@ func getAttachedInterfacesByID(client *gophercloud.ServiceClient, serviceID stri
 
 // Clusters is a no-op
 func (os *OpenStack) Clusters() (cloudprovider.Clusters, bool) {
+	err := os.ensureCloudProviderWasInitialized()
+	if err != nil {
+		return nil, false
+	}
+
```

Comment:
why does it matter to initialize here?

Reply:
yep, I did it for consistency, but there is no need to initialize here.